wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests/content.txt

```diff
@@ -44,10 +44,11 @@ The main commands/keywords (i.e. public methods) offered by !-nl.hsac.fitnesse.f
 The commands/keywords (i.e. public methods) added by !-nl.hsac.fitnesse.fixture.slim.JsonHttpTest-! are listed below.
 
 -|Comment|
-|post values as json to <url>      |Sends a POST containing all values (as JSON properties) to the url.                                                          |
-|json path <expression>            |Evaluates the (definite) expression against the last response received.                                                      |
-|json path count <expression>      |Evaluates the (indefinite) expression against the last response received, counting the number of results.                    |
-|all json path matches <expression>|Evaluates the expression against the last response received and returns all results. Intended to be used with 'show' command.|
+|post values as json to <url>                   |Sends a POST containing all values (as JSON properties) to the url.                                                                            |
+|json path <expression>                         |Evaluates the (definite) expression against the last response received.                                                                        |
+|json path count <expression>                   |Evaluates the (indefinite) expression against the last response received, counting the number of results.                                      |
+|all json path matches <expression>             |Evaluates the expression against the last response received and returns all results. Intended to be used with 'show' command.                  |
+|set json path <expression> to <value> in <json>|Evaluates the expression against the provided json and -if exactly one key is found- replaces that key's existing value with the provided value|
```

Comment:
In my case, we had a flow where the front-end gets a viewObject from the rest-service. The user uses a form in the front-end to update this data and post it to a json endpoint. 

In order to test back end field validations, we want to update the json and send it back, so that we can create test cases for the rest service without being dependent on the front-end.

In most cases it'l be possible to predefine the messages, but this approach leaves less clutter in the tests and is less sensitive to small changes in the view-object. 
In this case however, we needed to update a field with a value that depended on the previous value (version +1). In that case it's convenient to jus be able to update the reponse and use the updated response as the next request body.